src/styles/styles.js

```diff
@@ -31,6 +32,8 @@ import Colors from './colors';
 // touchCallout is an iOS safari only property that controls the display of the callout information when you touch and hold a target
 const touchCalloutNone = Browser.isMobileSafari() ? {WebkitTouchCallout: 'none'} : {};
 
+const lineHeightBadge = getPlatform() === CONST.PLATFORM.WEB ? {lineHeight: variables.lineHeightXSmall} : {lineHeight: variables.lineHeightNormal};
```

Comment:
Can we add a comment here about why we are adding this line? Ideally we discourage platform specific code and so once this is fixed upstream we can ideally come back and remove these changes

Reply:
As both @thienlnam and I agree that we should address the safari in specific (per accepted proposal) could you change this check to consist of `Browser.isSafari` ?